src/styles/styles.css

```diff
@@ -353,9 +353,8 @@ ul li.selected {
 .text-content a {
   @apply text-blue-600 underline hover:text-blue-800;
 }
-.text-content p,
-.search-results p {
-  @apply my-5;
+.text-content p + p {
```

Comment:
This may be naïve, but the intent here is to prevent leading and trailing margins on paragraphs. The issue is that creates gaps that affect the layout of neighbouring elements, which is undesirable.

Reply:
Makes sense, prefer this to `:first-child { @apply mt-0; }` !